index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Youtube video player</title>
+    <!-- adding css styles -->
+    <style>
+        body {
+            background-color: black;
+            color: rgb(231, 217, 217);
+            text-align: center;
+        }
+    
+        header,
+        main,
+        aside {
+            padding: 20px;
+            margin: 10px auto;
+            max-width: 800px;
+            background-color: #070707;
+            border-radius: 10px;
+        }
+    
+        iframe {
+            display: block;
+            margin: 0 auto;
+        }
+    
+        h2 {
+            font-size: 24px;
+            font-weight: bold;
+            margin-bottom: 10px;
+        }
+    
+        a {
+            color: red;
+            text-decoration: none;
+            display: block;
+            margin-bottom: 5px;
+        }
+    
+        a:hover {
+            color: #050505;
+            background-color: rgb(6, 6, 7);
+        }
+    </style>
+</head>
+
+<body>
+    <header>
+        <!--navigation bar-->
+        <div>
+            <nav> <a href="#"> Youtube Video Player</a></nav>
+            <nav> <a href="#"> home</a></nav>
+            <nav> <a href="#"> Subscribe </a></nav>
+            <nav> <a href="#"> related videos</a></nav>
+            <form> <input type="text" placeholder="search">
+                <button type="submit">search</button>
+        
+            </form>
+        
+        </div>
```

Comment:
Great job on using the header element to separate the head section of your HTML code from the rest of the body elements. I would suggest doing the same for each block of your code, by putting for example a section element <section></section> and giving it a class name so it would be easier to style it with CSS and you would have a much easier understanding of the code sections so when you want to update it, it would be much easier for you.